src/data/payments/captures/CaptureHelper.ts

```diff
@@ -41,4 +42,16 @@ export default class CaptureHelper extends Helper<CaptureData, Capture> {
     if (renege(this, this.getShipment, ...arguments)) return;
     return runIf(this.links.shipment, ({ href }) => this.networkClient.get<ShipmentData, Shipment>(href)) ?? undefinedPromise;
   }
+
+  /**
+   * Returns the shipment this capture has been settled with (if any).
```

Comment:
Should this be "settlement" instead of "shipment"?

```suggestion
   * Returns the settlement this capture has been settled with (if any).
```